frontend/src/constants/environments.ts

```diff
@@ -41,4 +42,5 @@ export const environment: { [key: string]: string } = {
   NEXT_BUILD: NEXT_BUILD || "false",
   SESSION_SECRET: SESSION_SECRET || "",
   NEXT_PUBLIC_BASE_URL: NEXT_PUBLIC_BASE_URL || "http://localhost:3000",
+  API_JWT_PUBLIC_KEY: API_JWT_PUBLIC_KEY || "",
```

Comment:
Thinking out loud, might be useful to error or write debug statements here if certain envars are missing.

Reply:
one problem there is that these will always be undefined in the build process, so we'd be polluting the logs there unless we check the NEXT_BUILD env var to gate the logging. We're only setting that flag in the Dockerfile at the moment, though. Worth a further discussion